eng/automation/alpine-env-setup.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
Why do you need this script?

Reply:
There is another feature to provide a docker to service team to generate all language LLC SDKs. The base image of the docker is Alpine, so the env-setup script is for that feature.